README.md

```diff
@@ -51,7 +64,7 @@ https://github.com/learn-co-students/forks-and-clones-readme-v-000
 
 [More on forking in the GitHub docs.](https://help.github.com/enterprise/2.2/user/articles/fork-a-repo/)
 
-## Cloning
+## Use `git clone` to Save a Repo to Your Local Machine
```

Comment:
Sticky verbs again. Saving is closer to commit. 

Maybe clone ==> download and fork ===> duplicate a repo on a remote host so that you own it?